qiskit/pulse/macros.py

```diff
@@ -74,15 +74,16 @@ def measure(qubits: List[int],
                 "argument. For assistance, the instructions which are defined are: "
                 "{}".format(measure_name, inst_map.instructions)) from ex
         for time, inst in default_sched.instructions:
+            if inst.channel.index not in qubits:
+                continue
             if qubit_mem_slots and isinstance(inst, instructions.Acquire):
                 if inst.channel.index in qubit_mem_slots:
                     mem_slot = channels.MemorySlot(qubit_mem_slots[inst.channel.index])
                 else:
                     mem_slot = channels.MemorySlot(unused_mem_slots.pop())
-                schedule = schedule.insert(time, instructions.Acquire(
-                    inst.duration, inst.channel, mem_slot=mem_slot))
-            elif qubit_mem_slots is None and isinstance(inst, instructions.Acquire):
-                schedule = schedule.insert(time, inst)
+                if inst.channel.index in qubits:
```

Comment:
Can remove this line

Reply:
Fixed in https://github.com/Qiskit/qiskit-terra/pull/5971/commits/65a3fc296ab5ed9747ef3df78aaf1f82e7741f7c